toml/test.ts

```diff
@@ -711,3 +711,38 @@ aaaaa = 1
     assertEquals(actual, expected);
   },
 });
+
+Deno.test({
+  name: "[TOML] stringify empty key",
+  fn() {
+    const src = {
+      "": "a",
+      "b": { "": "c" },
+    };
+    const roundTrip = parse(stringify(src));
+    assertEquals(src, roundTrip);
```

Comment:
I think we should directly assert the stringified result, instead of parsing it back to js object